packages/v4/src/content/design-guidelines/components/notification-drawer/notification-drawer.md

```diff
@@ -78,11 +78,11 @@ The notification drawer provides a flexible container where users can view and m
 
 When the notification drawer is being used, notifications will undergo the following life-cycle:
```

Comment:
This brings up a question I have, which is - should notification drawers be used instead of toast alerts, or along with them?
I wonder about situations that are just to confirm an action was done successfully. For example, if a user submits a modal, and you want to give them an alert to say "changes successfully saved" - would that be shown as a toast, or would it be stored in the alerts?

Reply:
You may choose to both use a toast alert at the same time as adding to the drawer or not depending on the circumstances. In the case you offered, I probably would not store this in the drawer unless it was important to track a list of all activity in the system. I added a paragraph at the end of the doc to attempt to advise on this. See what you think @mmenestr .